README.md

```diff
@@ -74,6 +74,13 @@ There are canonical examples defined in the class definition Javadoc of the clie
 Documentation can be found at the [Java Client
 Github Project Page](http://prometheus.github.io/client_java).
 
+### Disabling `_created` metrics
+
+By default, counters, histograms, and summaries export an additional series
```

Comment:
I know this is copy-and-paste from `client_python`. However, I would suggest replacing "By default, ..." with "In OpenMetrics format, ...".

Reply:
That may contradict both code and tests. The `_created` metrics seem to be emitted regardless of format (EA both 004 and 100 text formats. For example https://github.com/prometheus/client_java/blob/7c9fc397c21ebc9119f40131f075e1b2ed7b3079/simpleclient_common/src/test/java/io/prometheus/client/exporter/common/TextFormatTest.java#L57 .

Reply:
Oh, sorry, I forgot that `client_java` adds the `_created` series to text format as well.